extensions/toggl-track/src/menuBar.tsx

```diff
@@ -92,8 +85,8 @@ export default function Command() {
         <MenuBarExtra.Item
           title="Start/Stop Time Entry"
           icon={"command-icon.png"}
-          onAction={() =>
-            launchCommand({
+          onAction={async () =>
+            await launchCommand({
               name: "index",
               type: LaunchType.UserInitiated,
             })
```

Comment:
I think this will make more sense from a user perspective.
```suggestion
          onAction={async () => {
              if (runningEntry) await stopRunningTimeEntry(runningEntry);
              else await launchCommand({
                  name: "index",
                  type: LaunchType.UserInitiated,
              })
          }
```

Reply:
I don't think so. 

This command opens the the Raycast window where you can see the running time entry + the recent ones + ability to create new time entry. 

Opening that window from Raycast itself doesn't stop your running time entry, so this shouldn't either.